webapp/components/interop-dashboard.js

```diff
@@ -613,12 +613,16 @@ class InteropDashboard extends PolymerElement {
     return feature === this.feature;
   }
 
-  featureLinks(feature) {
+  featureLinks(feature, stable) {
     const data = this.getYearProp('focusAreas')[feature];
+    let testsURL = data?.tests;
```

Comment:
This section will set testsURL to undefined if `data?` is undefined. However down below in `formatTestsURL`, if data exists but `data.tests` does not exist, you return an empty string [here](https://github.com/web-platform-tests/wpt.fyi/pull/3287/files#diff-770c7c8879ccc187effa2395cb35a77882a85b9eb65fc536ae0545552e74a1f1R641). Maybe it would be nice to have a constant at the top like `EMPTY_URL = ''`;
```suggestion
    const testsURL = data?.tests : this.formatTestsURL(data?.tests, stable) : EMPTY_URL;
```

and 
```js
  formatTestsURL(testsURL, stable) {
    // Don't try to add a label if the URL is undefined or empty.
    if (!testsURL) {
      return EMPTY_URL;
    }
```

This way you're returning the same data type in the event there's missing data and the dom-if [here](https://github.com/web-platform-tests/wpt.fyi/pull/3287/files#diff-770c7c8879ccc187effa2395cb35a77882a85b9eb65fc536ae0545552e74a1f1R496) will work as expected.

Reply:
I agree that there's data type ambiguity here with my implementation like you mentioned. I removed the logic check and always invoke `formatTestsURL()` instead of checking if `testsURL` is valid beforehand, which will now always return an empty string if it isn't valid from `formatTestsURL()`. This should fix the problem. Let me know if you think this is a good approach!

Reply:
That works!